lib/youtube-dl.js

```diff
@@ -305,3 +305,21 @@ ytdl.getSubs = function(url, options, callback) {
     callback(null, files);
   });
 };
+
+/**
+ * @params {Boolean} descriptions
+ * @param {Function(!Error, Object)} callback
+ */
+ytdl.getExtractors = function(descriptions, options, callback) {
```

Comment:
should the `options` argument be there? I'm not sure that there are any other options that can be used in youtube-dl when extractors are listed.